x-pack/legacy/plugins/index_lifecycle_management/server/lib/license_pre_routing_factory/license_pre_routing_factory.js

```diff
@@ -5,8 +5,8 @@
  */
 
 import { once } from 'lodash';
-import { wrapCustomError } from '../error_wrappers';
 import { PLUGIN_ID } from '../../../common/constants';
+import { wrapCustomError } from '../error_wrappers';
 
 export const licensePreRoutingFactory = once(server => {
```

Comment:
Just an FYI, I just opened a PR for adding examples of how to migrate these pre-route handlers: #56080. Not relevant for this PR, but should be helpful for completing the server-side.